src/main/java/de/siphalor/modsoftheworld/client/MOTWClient.java

```diff
@@ -10,14 +24,6 @@
 import net.minecraft.resource.ResourceManager;
 import net.minecraft.util.Identifier;
 import net.minecraft.util.InvalidIdentifierException;
-import org.lwjgl.opengl.GL32;
-
-import java.io.BufferedReader;
-import java.io.IOException;
-import java.io.InputStreamReader;
-import java.nio.charset.StandardCharsets;
-import java.util.*;
-import java.util.regex.Pattern;
```

Comment:
eclipse auto format, I'd normally avoid pushing this to a branch except it resolved the wildcard imports which is best practice imho